ginga/util/catalog.py

```diff
@@ -63,21 +46,19 @@ def has_key(self, key):
         return key in self.starInfo
 
 
-# TODO: Deprecate this class name and replace with something that reflects
-#       Cone Search in Astroquery.
-class AstroPyCatalogServer(object):
+class AstroQueryVOCatalogServer(object):
```

Comment:
This renaming is certainly not backward-compatible?
```suggestion
class AstroqueryVOCatalogServer(object):
```